ingest/sqs/README.md

```diff
@@ -11,6 +11,276 @@ The use case for this is performing certain data and computationally intensive t
 the chain node or the clients. For example, routing falls under this category because it requires
 all pool data for performing the complex routing algorithm.
 
+## Supported Endpoints
+
+## Pools Resource
+
+1. GET `/pools/all`
+
+Description: returns all pools in the chain state instrumented with denoms and TVL if available
+
+Parameters: none
+
+Response example:
+```bash
+curl "https://sqs.osmosis.zone/pools/all" | jq .
+[
+  {
+    "underlying_pool": {
+      "address": "osmo164pg0key096kxe7940h45csw5w9cmf4nc2a83m73e8tc0u2eymeqj7t0rm",
+      "incentives_address": "osmo1z6ae7saxlwnnd6whl2svw0k3zhsjdvzaap9ya2lcj8m3c0r02a6svtuzn2",
+      "spread_rewards_address": "osmo1jhzypnueeg75cm8tps7srn7ujva9qe4msjmtkajwaqpj0kyvdjjqu0xule",
+      "id": 1323,
+      "current_tick_liquidity": "1000000.000000000000100000",
+      "token0": "ibc/ECBE78BF7677320A93E7BA1761D144BCBF0CBC247C290C049655E106FE5DC68E",
+      "token1": "uosmo",
+      "current_sqrt_price": "1.000000000000000000000000000000000000",
+      "tick_spacing": 100,
+      "exponent_at_price_one": -6,
+      "spread_factor": "0.000500000000000000",
+      "last_liquidity_update": "2023-12-06T14:36:29.772040341Z"
+    },
+    "sqs_model": {
+      "total_value_locked_uosmo": "1000000",
+      "total_value_locked_error": "error getting token precision ibc/ECBE78BF7677320A93E7BA1761D144BCBF0CBC247C290C049655E106FE5DC68E",
+      "balances": [
+        {
+          "denom": "ibc/ECBE78BF7677320A93E7BA1761D144BCBF0CBC247C290C049655E106FE5DC68E",
+          "amount": "1000000"
+        },
+        {
+          "denom": "uosmo",
+          "amount": "1000000"
+        }
+      ],
+      "pool_denoms": [
+        "ibc/ECBE78BF7677320A93E7BA1761D144BCBF0CBC247C290C049655E106FE5DC68E",
+        "uosmo"
+      ],
+      "spread_factor": "0.000500000000000000"
+    }
+  },
+  ...
+]
+```
+
+## Router Resource
+
+
+1. GET `/router/quote?tokenIn=<amount><denomIn>&tokenOutDenom=<denomOut>`
+
+Description: returns the best quote it can compute for the given tokenIn and tokenOutDenom
+
+Parameters:
+- `amountIn` integer amount of tokenIn
+- `denomIn` string denom of tokenIn
+- `denomOut` string denom of tokenOut
+
+Response example:
+
+```bash
+curl "https://sqs.osmosis.zone/router/quote?tokenIn=1000000uosmo&tokenOutDenom=uion" | jq .
```

Comment:
This curl doesn't appear to match the parameters above. Also description doesn't match params

Reply:
```
/router/quote?tokenIn=<amount><denomIn>&tokenOutDenom=<denomOut>
```

```
https://sqs.osmosis.zone/router/quote?tokenIn=1000000uosmo&tokenOutDenom=uion
```

- `amountIn` integer amount of tokenIn
- `denomIn` string denom of tokenIn
- `denomOut` string denom of tokenOut

where:
- `amountIn` is 1000000
- `denomIn` is uosmo
- `denomOut` is uosmo

Reply:
Noticed related issues and fixed here:
[e7e48d1](https://github.com/osmosis-labs/osmosis/pull/7026/commits/e7e48d12d02409f6192daff73d268d40b75f5342)

Reply:
I see the confusion, please check this update:
[fcb6cec](https://github.com/osmosis-labs/osmosis/pull/7026/commits/fcb6cecaf924d3a228afcc457c36ad267e0373b9)

Reply:
OK that's what I had before

Reply:
OK that's what I had before